src/_data/architecture/sdp.yml

```diff
@@ -386,4 +386,12 @@ vendors:
     license: "Open Source"
     deployment: "Self-hosted"
     pricing: "n/a"
-    notes: ""
\ No newline at end of file
+    notes: ""
+
+  - company: "Timus Networks"
+    product: "Timus Networks"
+    url: "https://timusnetworks.com"
+    license: "Commercial"
+    deployment: "SaaS"
+    pricing: "https://www.timusnetworks.com/plans/"
```

Comment:
Pricing should be either `Published`, `Not Published` or `n/a` if the product is non-commercial. Please could you change this to `Published` if that's correct?